poppy/wfe.py

```diff
@@ -356,7 +331,7 @@ def __init__(self, name='PSD WFE', index=3.0, wfe=50*u.nm, radius=1*u.meter, see
         self.radius = radius
         self.seed = seed
 
-    @_accept_wavefront_or_meters
+    @_check_wavefront_arg
     def get_opd(self, wave, units='meters'):
```

Comment:
Did you also want to get rid of the units parameter for this method?

Reply:
oh good catch, I think yes.

Reply:
removed.